.github/workflows/branch_track_automation.yaml

```diff
@@ -1,9 +1,6 @@
-name: Branch and Track Creation Automation
+name: Branch Creation Automation
 
 on:
-  push:
```

Comment:
Please omit this comment if I am wrong, but isn't this what we want? After a PR is merged, it will "push to main", which will then trigger this workflow.

Reply:
Not anymore. Since we decided that bundle-kubeflow would be the source of truth for bundle yamls, this action should not be triggered on push (in kubeflow-ci). I opened a pr in bundle-kubeflow to trigger this workflow from bundle-kubeflow's on push

Reply:
True, sorry, I was thinking on the other charm repos.